File: 1344-angle-between-hands-of-a-clock/1344-angle-between-hands-of-a-clock.cpp
class Solution {
public:
    double angleClock(int hour, int minutes) {
        double mint = 6*minutes;
        double h = (double)(30*hour%360) + (double)(0.5)*minutes;
        double angle = abs(mint-h);

        return min(angle,abs(angle-360));
    }
};

File: 1344-angle-between-hands-of-a-clock/1344-angle-between-hands-of-a-clock.java
class Solution {
    public double angleClock(int hour, int minutes) {
        double h = (double)(hour*30 %360) + (0.5*minutes);
        double mint = 6*minutes;

        double diff = Math.abs(mint-h);

        return Math.min(diff,360-diff);
    }
}